libs/langchain/langchain/document_loaders/url_selenium.py

```diff
@@ -115,6 +115,37 @@ def _get_driver(self) -> Union["Chrome", "Firefox"]:
         else:
             raise ValueError("Invalid browser specified. Use 'chrome' or 'firefox'.")
 
+    def _build_metadata(self, url: str, driver: Union["Chrome", "Firefox"]) -> dict:
```

Comment:
Note for reviewer: I'm actually not sure this is correct. I'm simply copying the return type from _get_driver, but I'm wondering if these quote should actually be here.

Reply:
Follow-up, engineers much smarter than me have confirmed that putting quoted strings inside of a Union doesn't mean the literal string, but "A class of this name to be defined later"